hack/kiali/func-sm.sh

```
@@ -0,0 +1,435 @@
+  #while [ "$(${OC} get validatingwebhookconfigurations -o name | grep -E 'sail|servicemesh|istio')" == "" ]; do echo -n '.'; sleep 5; done
+  #infomsg "done."
+  #
+  #infomsg "Wait for the servicemesh mutating webhook to be created."
+  #while [ "$(${OC} get mutatingwebhookconfigurations -o name | grep -E 'sail|servicemesh|istio')" == "" ]; do echo -n '.'; sleep 5; done
+  #infomsg "done."
+
+  # "latest" is not a supported version when using a released version of Sail operator.
+  # We try to determine the latest version of Istio supported by examining the CRD.
+  if [ "${istio_version}" == "latest" ]; then
+    istio_version="$(${OC} get crd istios.sailoperator.io -o json | jq -r '.spec.versions | sort_by(.name) | .[-1].schema.openAPIV3Schema.properties.spec.properties.version.default')"
+    if [ -z "${istio_version}" -o "${istio_version}" == "null" ]; then
```

Shellcheck (0.11.0)

[warning] Prefer [ p ] || [ q ] as [ p -o q ] is not well defined.

(SC2166)